### PR TITLE
8366 Misc fixes to certify section for coops

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#8366

*Description of changes:*
- added resource property for entity display string
- removed extra colon from resource title
- replaced ’ with '
- replaced some ` with '
- fixed line indents
- fixed coop section and act
- app version: 2.0.7

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/resources/benefitCompany.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module
 export const BenefitCompanyResource: ResourceIF = {
   entityType: CorpTypeCd.BENEFIT_COMPANY,
   displayName: GetCorpFullDescription(CorpTypeCd.BENEFIT_COMPANY),
-  title: 'Benefit Company Statement:',
```
Comment (severinbeauvais) [inline src/resources/benefitCompany.ts:9]: This caused a double colon.

File: src/resources/benefitCompany.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@ import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module
...
-  description: `This company is a benefit company and, as such, has purposes that include conducting its business
-        in a responsible and sustainable manner and promoting one or more public benefits.`,
```
Comment (severinbeauvais) [inline src/resources/benefitCompany.ts:11]: Using back-ticks for multi-line strings is neat, but while testing, the whitespace (including indents) must match exactly, forcing us to use weird indents in some places. So I changed this to concatenated individual strings in a few places.

File: src/resources/benefitCompany.ts [hunk 3]
```diff
@@ -29,17 +29,17 @@ export const BenefitCompanyResource: ResourceIF = {
         helpText: [
           `The sample Incorporation Agreement and Benefit Company Articles is a template that you can use
             to create an incorporation agreement and articles for your company. It uses all the standard
-            provisions suggested by legislation and also includes a place to specify the company’s benefit
```
Comment (severinbeauvais) [inline src/resources/benefitCompany.ts:32]: This apostrophe is fine in a word processor but its use is inconsistent here, so I changed it to the usual apostrophe (in a few places).

File: src/resources/benefitCompany.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/resources/benefitCompany.ts:9]: image

Comment (severinbeauvais) [reply src/resources/benefitCompany.ts:9]: image

**Review by argush3 — APPROVED** [review]
LGTM

**Pull request merged**